### PR TITLE
Colorize log messages on create, modify and delete

### PR DESCRIPTION
**Pull request by mozamimy**
I think it's better to colorize create, modify and delete logs. Especially we should warn enough about deleting some resources.

@wata-gh please review 🙏

### CODE REVIEW

**Comment by mozamimy** [comment]
ping @wata-gh

File: simnos.gemspec [hunk 1]
```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'hashie'
   spec.add_dependency 'diffy'
   spec.add_dependency 'term-ansicolor'
```
Comment (wata-gh) [inline simnos.gemspec]: I was thinking to use term-ansicolor to colorize logs.
So, if you use colorize gem there is no reason to have a dependency for term-ansicolor.
Can you remove this line too?
Thanks for your support!

Comment (mozamimy) [reply simnos.gemspec]: Removed the line in https://github.com/codenize-tools/simnos/pull/10/commits/0ca073c59282de021b1b57907a8839d53513c031 .

I'll rebase the commit before merging.

**Review by wata-gh — APPROVED** [review]

**Comment by wata-gh** [comment]
@mozamimy 
Will you rebase this pull-request or should I merge it right now?

**Comment by mozamimy** [comment]
I have rebased this branch. Thanks for your reviewing!

**Comment by wata-gh** [comment]
Thank you!

**Pull request merged**